systems/data-plan/package/pkg/server/package_test.go

```diff
@@ -55,24 +55,24 @@ func TestPackageServer_GetPackages_Error1(t *testing.T) {
 // Get Package by org //
 
 func TestPackageServer_GetPackageByOrg_Success(t *testing.T) {
-	var orgId = uuid.New()
+	var orgId = uuid.NewV4()
 
 	packageRepo := &mocks.PackageRepo{}
 	var mockFilters = &pb.GetByOrgPackageRequest{
-		OrgId: orgId.String(),
+		OrgID: orgId.String(),
 	}
-	s := NewPackageServer(packageRepo)
+	s := NewPackageServer(packageRepo, nil)
 
 	packageRepo.On("GetByOrg", orgId).Return([]db.Package{{
-		Sim_type:     pb.SimType_INTER_MNO_ALL.String(),
-		Name:         "Daily-pack",
-		Org_id:       orgId,
-		Active:       true,
-		Duration:     1,
-		Sms_volume:   20,
-		Data_volume:  12,
-		Voice_volume: 34,
-		Org_rates_id: 00,
+		SimType:     1,
```

Comment:
Using the type enum (instead of the number) will make it easier to update/debug in the future

Reply:
addressed